examples/first-component/src/components/ScientistList/ScientistsList.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import ScientistList from "./index";
+const meta: Meta<typeof ScientistList> = {
+  component: ScientistList,
+};
+
+export default meta;
+type Story = StoryObj<typeof ScientistList>;
+const scientist = [
```

Comment:
`scientist` is a single noun. A list should have naming plural.

Reply:
Fixed